src/pages/Lend/AlgoliaPagination.vue

```diff
@@ -1,103 +1,155 @@
 <template>
-	<ais-pagination :padding="padding">
-		<ul class="ais-Pagination-list"
-			slot-scope="{
-			currentRefinement,
-			nbPages,
-			pages,
-			isFirstPage,
-			isLastPage,
-			refine,
-			createURL
-			}"
+	<ul class="ais-Pagination-list">
+		<!-- previous page -->
+		<li
+			class="ais-Pagination-item ais-Pagination-item--previousPage"
+			:class="isFirstPage ? 'ais-Pagination-item--disabled' : ''">
```

Comment:
Let's use object syntax for this
https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax
`:class="{'ais-Pagination-item--disabled': isFirstPage}"`

Reply:
@ksekhri PR updated with object syntax.